docs/BUILD.bazel

```diff
@@ -3,46 +3,97 @@ load("@aspect_bazel_lib//lib:docs.bzl", "stardoc_with_diff_test", "update_docs")
 stardoc_with_diff_test(
     name = "js_binary",
     bzl_library_target = "//js/private:js_binary",
+    # https://github.com/bazelbuild/stardoc/pull/141
+    target_compatible_with = select({
+        "//:bzlmod": ["@platforms//:incompatible"],
+        "//conditions:default": [],
+    }),
 )
 
 stardoc_with_diff_test(
     name = "js_library",
     bzl_library_target = "//js/private:js_library",
+    # https://github.com/bazelbuild/stardoc/pull/141
+    target_compatible_with = select({
+        "//:bzlmod": ["@platforms//:incompatible"],
+        "//conditions:default": [],
+    }),
 )
 
 stardoc_with_diff_test(
     name = "js_run_binary",
     bzl_library_target = "//js/private:js_run_binary",
+    # https://github.com/bazelbuild/stardoc/pull/141
+    target_compatible_with = select({
+        "//:bzlmod": ["@platforms//:incompatible"],
+        "//conditions:default": [],
+    }),
 )
 
 stardoc_with_diff_test(
     name = "js_run_devserver",
     bzl_library_target = "//js/private:js_run_devserver",
+    # https://github.com/bazelbuild/stardoc/pull/141
+    target_compatible_with = select({
+        "//:bzlmod": ["@platforms//:incompatible"],
+        "//conditions:default": [],
+    }),
 )
 
 stardoc_with_diff_test(
     name = "js_filegroup",
     bzl_library_target = "//js/private:js_filegroup",
+    # https://github.com/bazelbuild/stardoc/pull/141
+    target_compatible_with = select({
+        "//:bzlmod": ["@platforms//:incompatible"],
+        "//conditions:default": [],
+    }),
 )
 
 stardoc_with_diff_test(
     name = "js_image_layer",
     bzl_library_target = "//js/private:js_image_layer",
+    target_compatible_with = select({
+        "//:bzlmod": ["@platforms//:incompatible"],
+        "//conditions:default": [],
+    }),
 )
 
 stardoc_with_diff_test(
     name = "npm_package",
     bzl_library_target = "//npm/private:npm_package",
+    # https://github.com/bazelbuild/stardoc/pull/141
+    target_compatible_with = select({
+        "//:bzlmod": ["@platforms//:incompatible"],
+        "//conditions:default": [],
+    }),
 )
 
 stardoc_with_diff_test(
     name = "npm_link_package",
     bzl_library_target = "//npm/private:npm_link_package",
+    # https://github.com/bazelbuild/stardoc/pull/141
+    target_compatible_with = select({
+        "//:bzlmod": ["@platforms//:incompatible"],
+        "//conditions:default": [],
+    }),
 )
 
 stardoc_with_diff_test(
     name = "npm_import",
     bzl_library_target = "//npm:npm_import",
+    # https://github.com/bazelbuild/stardoc/pull/141
```

Comment:
this is a lot of boilerplate, maybe it could go in a wrapper macro or even in bazel-lib where this symbol is loaded?

Reply:
Added a macro